setup.py

```diff
@@ -24,7 +24,7 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere-sagemaker',
-                 version='0.7.2',
+                 version='0.8',
```

Comment:
Should be 0.8.0 right? Please remember to publish to pypi

Reply:
Will publish to pypi once merged 👍